tests/vlan/test_vlan.py

```diff
@@ -321,6 +404,12 @@ def verify_icmp_packets(ptfadapter, vlan_ports_list, vlan_port, vlan_id):
                                     portchannel_ports=tagged_dst_pc_ports)
 
 
+def check_expected_packet_result(exp):
```

Comment:
>check_expected_packet_result [](http://example.com/codeflow?start=4&length=28)

You can even add `testutils.verify_packets_any()` to this fuction, and make caller only one line.